package.json

```diff
@@ -2,9 +2,29 @@
   "name": "linter-luacheck",
   "description": "Lint Lua on the fly, using luacheck",
   "keywords": [
+    "lint",
     "linter",
-    "lua"
+    "lua",
+    "luacheck",
+    "lua-lint"
   ],
+  "configSchema": {
+    "executable": {
+      "type": "string",
+      "default": "luacheck",
+      "description": "The executable path to luacheck."
+    },
+    "globals": {
+      "type": "array",
+      "default": [],
+      "description": "Add follow comma separated globals on top of standard ones. eg. `jit, bit`"
```

Comment:
That is actually what was already there... it sounded awful so I was gonna rewrite it later on but guess that I'll do it!